features/cerberusweb.core/api/plugin.classes.php

```diff
@@ -290,16 +298,30 @@ function run($token, Model_TriggerEvent $trigger, $params, DevblocksDictionaryDe
 		if(empty($response_placeholder))
 			return false;
 		
-		$response = $this->_execute($http_verb, $http_url, array(), $http_body, $http_headers, $options);
+		if(!empty($http_cert)) {
+			if(!file_exists($http_cert)) {
+				return "[ERROR] HTTPS client certificate file not found: $http_cert";
+			}
+		}
+
+		$response = $this->_execute($http_verb, $http_url, array(), $http_body, $http_headers, $options, $http_cert);
 		$dict->$response_placeholder = $response;
 	}
 	
-	private function _execute($verb='get', $url, $params=array(), $body=null, $headers=array(), $options=array()) {
+	private function _execute($verb='get', $url, $params=array(), $body=null, $headers=array(), $options=array(), $http_cert=null) {
 		if(!empty($params) && is_array($params))
 			$url .= '?' . http_build_query($params);
 		
 		$ch = DevblocksPlatform::curlInit($url);
 		
+		// explicitly set the SSL Cert to null when no http_cert passed, so that the underlying
+		// extension flushes its cache.
+		if(empty($http_cert)){
```

Comment:
The IF/Else is not strictly required, but I left it in to be clear that the CURLOPT_SSLCERT needs to be unset, or a previously set CURLOPT_SSLCERT value is cached/remembered by the PHP-CURL extension